luminoth/tools/cloud/gcloud.py

```diff
@@ -232,13 +263,96 @@ def train(job_id, service_account_json, bucket_name, region, config_files,
         click.echo('Job {} submitted successfully.'.format(job_id))
         click.echo('state = {}, createTime = {}'.format(
             res.get('state'), res.get('createTime')))
+
+        save_run(config, environment='gcloud', extra_config=job_spec)
+
     except Exception as err:
         click.echo(
             'There was an error creating the training job. '
             'Check the details: \n{}'.format(err._get_reason())
         )
 
 
+@gc.command(help='Start a evaluation job')
+@click.option('--job-id', required=True, help='JobId for saving models and logs.')  # noqa
+@click.option('--service-account-json', required=True)
+@click.option('--bucket', 'bucket_name', help='Where to save models and logs.')  # noqa
+@click.option('dataset_split', '--split', default='val', help='Dataset split to use.')  # noqa
+@click.option('--region', default='us-central1', help='Region in which to run the job.')  # noqa
+@click.option('--scale-tier', default=DEFAULT_SCALE_TIER, type=click.Choice(SCALE_TIERS))  # noqa
+@click.option('--rebuild', default=False, is_flag=True, help='Rebuild and upload package.')  # noqa
+@click.option('--postfix', default='eval', help='Postfix for the evaluation job name.')  # noqa
+def evaluate(job_id, service_account_json, bucket_name, dataset_split, region,
+             scale_tier, rebuild, postfix):
+    project_id = get_project_id(service_account_json)
+    if project_id is None:
+        raise ValueError(
+            'Missing "project_id" in service_account_json "{}"'.format(
+                service_account_json))
+
+    if bucket_name is None:
+        client_id = get_client_id(service_account_json)
+        bucket_name = 'luminoth-{}'.format(client_id)
+        click.echo(
+            'Bucket name not specified. Using "{}".'.format(bucket_name))
+
+    credentials = get_credentials(service_account_json)
+    validate_region(region, project_id, credentials)
+    job_folder = 'lumi_{}'.format(job_id)
```

Comment:
lumi should be configurable.

Reply:
Not really, we don't really want to overload users with options. Maybe just remove it but that is outside the scope of this PR.